src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/LiquidViewFilters.cs

```diff
@@ -29,17 +37,17 @@ public static ValueTask<FluidValue> Localize(FluidValue input, FilterArguments a
         {
             if (!context.AmbientValues.TryGetValue("ViewLocalizer", out var localizer))
             {
-                throw new ArgumentException("ViewLocalizer missing while invoking 't'");
+                ThrowArgumentException("ViewLocalizer missing while invoking 't'");
```

Comment:
Now using helper that allows JIT to inline method when seems to be a good idea, methods with throw won't inline as that would change stack trace - here it would be fine.